hieroweb/src/main/webapp/#table.ts#

```diff
@@ -0,0 +1,930 @@
+/*
```

Comment:
You should not check-in temporary files. Maybe you can add a patter to .gitignore.
Please delete this file and the next one from the repository.